### PR TITLE
Filled out Signature['Element']

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

A continuation of #20.

The pull request uses the `Element` node to illustrate how, given `context.signature`, we can find and update the component's `Signature` in the `update-signature` step.

### CODE REVIEW

File: tests/fixtures/ember-container-query-glint/input/src/components/tracks.hbs [hunk 1]
```diff
@@ -8,7 +8,10 @@
   as |CQ|
 >
   {{#if (strict-and CQ.features.large CQ.features.tall)}}
-    <Tracks::Table @tracks={{@tracks}} />
+    <Tracks::Table
+      @tracks={{@tracks}}
+      ...attributes
+    />
```
Comment (ijlee2) [inline tests/fixtures/ember-container-query-glint/input/src/components/tracks.hbs:14]: This component covers an edge case of `Element`. There is dynamic code (resulting from an `{{if}}`-statement) as well as more than one (1) `...attributes`. It's hard to guarantee correctness.

https://github.com/emberjs/rfcs/blob/master/text/0748-glimmer-component-signature.md#components-with-multiple-or-varying-elements
